sass/components/html/file--link.html

```diff
@@ -1,8 +1,11 @@
 <div class="file file--link">
-  <span class="file__icon icon icon--file-link"></span>
```

Comment:
Ok, I'll need some coordinates on how to "check how do we output file--link(s) at the moment".